packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1736,6 +1737,10 @@ class Scaffold extends StatefulWidget {
   ///    to the top of the scaffold.
   final bool extendBody;
 
+  /// If false, and a [drawer] is specified, then the barrier behind the drawer should not respond to a tap event and thus remain open.
+  /// The default behavior is to close upon the user tapping on the barrier.
```

Comment:
There should be an empty line of `///` in between these two.

Also, check the 80 character line limit.